pycm/pycm_obj.py

```diff
@@ -218,6 +218,26 @@ def __iter__(self):
         for key in self.matrix.keys():
             yield key, self.matrix[key]
 
+    def __contains__(self, c):
+        """
+        Check if given the confusion matrix contains given class.
+
+        :param c: given class
+        :param c: Any
+        :return: True if c is a class in confusion matrix
+        """
+        return c in self.classes
+
+    def __getitem__(self, i):
+        """
+        Return i'th row of the confusion matrix.
+
+        :param i: given index
+        :type i: int
+        :return: i'th row of confusion matrix.
+        """
+        return self.to_array()[i]
```

Comment:
I think it would be better if we return items by `class_name` instead of index.
For example:
```
cm[class_name1][class_name2]
```

Reply:
So it's now simply a alias for
```
cm.matrix[class_name1][class_name2]
```

Reply:
Yeah! You are right.
But indexing doesn't make sense, is it?

Reply:
Since the object name is a confusion matrix I think it would actually make sense when we are indexing it like the table.

Reply:
When you don't know the order of rows (especially in large cases) I think it doesn't make sense. For example, a class name is `Dog` with index 1, in the first step user should print the confusion matrix or take a look at the `classes` list to find the index!
If you are not satisfied, we can talk about it in the next weekly session!

Reply:
You're right.
What do you think about trying to return using input as `class_name` and if it doesn't try to use numeric indexing.
Another feature we can have using numeric indexing would be `numpy`-like indexing which comes automatically when we use numeric indexing.

Reply:
How? Would you please give an example?

Reply:
```pycon
>>> cm = ConfusionMatrix(matrix={"Class1": {"Class1": 1, "Class2":2}, "Class2": {"Class1": 0, "Class2": 5}})
>>> cm['Class1']['Class1']
1
>>> cm[0][0]
1
>>> cm['Class1'][0]
ERROR
>>> cm[0]['Class1']
ERROR
```

Reply:
I'm not okay with this approach.
It's some kind of dual behavior

Reply:
@alirezazolanvari What's your opinion?

Reply:
I just implement your idea. If @alirezazolanvari is OK with it, too we can merge then.